src/frontend/screens/Invites/InviteSuccessfullyJoined.tsx

```diff
@@ -0,0 +1,80 @@
+import {BottomSheetWrapper} from '../../sharedComponents/BottomSheetWrapper';
+import {StyleSheet, View} from 'react-native';
+import {PrimaryButton, SecondaryButton} from '../../sharedComponents/Buttons';
+import {defineMessages, useIntl} from 'react-intl';
+import {NativeRootNavigationProps} from '../../sharedTypes/navigation';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText';
+import {BodyText} from '../../sharedComponents/Text/BodyText';
+import {CommonActions, useNavigation} from '@react-navigation/native';
+import SuccessCheck from '../../images/GreenCheck.svg';
+
+const m = defineMessages({
+  goToMap: {
+    id: 'screens.InviteSuccess.goToMap',
+    defaultMessage: 'Go To Map',
+  },
+  success: {
+    id: 'screens.InviteSuccess.success',
+    defaultMessage: 'Success',
+  },
+  youHaveJoined: {
+    id: 'screens.InviteSuccess.youHaveJoined',
+    defaultMessage: 'You have joined {projectName}',
+  },
+  goToSync: {
+    id: 'screens.InviteSuccess.goToSync',
+    defaultMessage: 'Go To Sync',
+  },
+});
+
+export const InviteSuccessfullyJoined = ({
```

Comment:
sorry for another nit but `InviteSuccessfullyAccepted` seems like a more accurate naming of what this is reflecting. maybe it's just me but I read this as someone tried to join an invite, which doesn't make sense to me 😅